src/power_grid_model_io/data_stores/excel_file_store.py

```diff
@@ -28,61 +28,112 @@ class ExcelFileStore(BaseDataStore[TabularData]):
 
     _unnamed_pattern: re.Pattern = re.compile(r"Unnamed: \d+_level_\d+")
 
-    def __init__(self, *file_paths: Path):
+    def __init__(self, file_path: Optional[Path] = None, **extra_paths: Path):
         super().__init__()
-        self._file_paths: List[Path] = list(file_paths)
+
+        # Create a list of all supplied file paths:
+        # [(None, file_path), [extra_name[0], extra_path[0]), [extra_name[1], extra_path[1]), ...]
```

Comment:
for the extra paths it is opened as a list `[` and closed as a tuple `)`. Guess it should be:
`[(None, file_path), (extra_name[0], extra_path[0]), (extra_name[1], extra_path[1]), ...]`

Reply:
Now read the lines below, it a dict:

`{None: file_path, extra_name[0]: extra_path[0], extra_name[1]: extra_path[1], ...}`